src/proxy/http/profiles.rs

```diff
@@ -16,7 +20,13 @@ use never::Never;
 use super::retry::Budget;
 use NameAddr;
 
-pub type Routes = Vec<(RequestMatch, Route)>;
+#[derive(Clone, Debug, PartialEq, Eq, Hash)]
+pub struct ConcreteDst(pub NameAddr);
+
+pub struct Routes {
+    pub routes: Vec<(RequestMatch, Route)>,
+    pub dst_overrides: Vec<(NameAddr, u32)>,
```

Comment:
Nit/TIOLI: What do you think about changing these tuples to structs? Code consuming this might read better with named field accesses rather than `.1`/`.0`.